tests/test_module.py

```diff
@@ -74,6 +74,22 @@ def test_p(self):
         assert module.p("param_1") == 5000.0
     """
 
+    def test_set_parameter(self):
+        synthglobals = torchsynth.globals.SynthGlobals(batch_size=T(2))
+        module = synthmodule.ADSR(synthglobals, attack=T([0.5, 1.0]))
```

Comment:
Just another note here, if floats actually are faster, then maybe we should consider a *separate* PR where we convert all the scalars to plain floats and see the performance changes?

Reply:
Lets do it.

Reply:
#239